File: src/main/java/net/atos/zac/app/informatieobjecten/model/RestGekoppeldeZaakEnkelvoudigInformatieObject.java
/*
 * SPDX-FileCopyrightText: 2022 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.app.informatieobjecten.model;

import java.util.UUID;

import jakarta.ws.rs.FormParam;

import net.atos.zac.app.zaak.model.RelatieType;

public class RestGekoppeldeZaakEnkelvoudigInformatieObject extends RestEnkelvoudigInformatieobject {

    @FormParam("relatieType")
    public RelatieType relatieType;

    @FormParam("zaakIdentificatie")
    public String zaakIdentificatie;

    @FormParam("zaakUUID")
    public UUID zaakUUID;
}

File: src/main/kotlin/nl/info/zac/app/informatieobjecten/model/RestGekoppeldeZaakEnkelvoudigInformatieObject.kt
/*
 * SPDX-FileCopyrightText: 2022 Atos, 2026 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */

package nl.info.zac.app.informatieobjecten.model

import nl.info.zac.app.zaak.model.RelatieType
import java.util.UUID

class RestGekoppeldeZaakEnkelvoudigInformatieObject : RestEnkelvoudigInformatieobject() {
    var relatieType: RelatieType? = null

    var zaakIdentificatie: String? = null

    var zaakUUID: UUID? = null
}
